cpp/arcticdb/CMakeLists.txt

```diff
@@ -509,6 +512,19 @@ endif ()
 
 target_link_libraries(arcticdb_core_object PUBLIC ${arcticdb_core_libraries})
 
+set (azure_package
+        Azure::azure-identity
+        Azure::azure-storage-blobs
```

Comment:
Just add these to the `arcticdb_core_libraries` list like *all* the other dependencies and get rid of the extra call to `target_link_libraries` below.